packages/tupaia-web/src/utils/getApiUrl.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export const getApiUrl = () => {
+  return process.env.REACT_APP_CONFIG_SERVER_BASE_URL || 'http://localhost:8080/api/v1/';
```

Comment:
```suggestion
  return process.env.REACT_APP_TUPAIA_WEB_API_URL || 'http://localhost:8100/api/v1/';
```
I think that's the standard .env var pattern now 👍